pom.xml

```diff
@@ -205,25 +205,19 @@
         <dependency>
             <groupId>io.github.bonigarcia</groupId>
             <artifactId>webdrivermanager</artifactId>
-            <version>1.7.2</version>
+            <version>3.3.0</version>
         </dependency>
 
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
             <version>1.19</version>
         </dependency>
+
         <dependency> <!-- Required for webdrivermanager logging -->
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-slf4j-impl</artifactId>
             <version>2.0.2</version>
         </dependency>
-
-        <!-- Download latest drivers -->
-        <dependency>
-            <groupId>io.github.bonigarcia</groupId>
```

Comment:
So this dependency is no longer necessary with the change in the import statement in the DriverUtil.java class (or for some other reason)?

Reply:
It was a duplication of the dependency of whose version I updated above (and cause no end of headaches when I couldn't figure out without assistance why a simple version update wasn't working).